bluepysnap/morph.py

```diff
@@ -52,26 +58,46 @@ def __init__(self, morph_dir, population):
     def _is_biophysical(self, node_id):
         return self._population.get(node_id, Node.MODEL_TYPE) == "biophysical"
 
-    def get_filepath(self, node_id):
+    def _get_morph_dir(self, extension):
+        """Return morphology directory based on a given extension."""
+        if extension == 'swc':
+            if not self._morph_dir:
+                raise BluepySnapError("'morphologies_dir' is not defined in config")
+            return self._morph_dir
+
+        alternate_key = EXTENSIONS_MAPPING.get(extension)
+        if not alternate_key:
+            raise BluepySnapError(f"Unsupported extension: {extension}")
+
+        morph_dir = self._alternate_morph_dir.get(alternate_key)
+        if not morph_dir:
+            raise BluepySnapError(f"'{alternate_key}' is not defined in 'alternate_morphologies'")
+
+        return morph_dir
+
+    def get_filepath(self, node_id, extension='swc'):
```

Comment:
I don't like this `extension` kwarg at all but having different functions for the alternate morphs did not make sense either. ArchNGV needs this to return a path, though. So, if anyone happens to come up with a better solution, please let me know and I'll happily change this one.